### PR TITLE
Conrad branch test

### PR DESCRIPTION
**Pull request by conradboucher**

### CODE REVIEW

**Comment by arishar** [comment]
this is great, i approve

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # llcpython
 # committing to the main branch as a test
+#this is a comment
```
Comment (cstolitzka) [inline README.md]: this needs to be updated to be capititalized

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
 
 This is a commit from Sid to test if git has been setup correctly on my laptop.
```
Comment (cstolitzka) [inline README.md]: love what you are doing!

File: README.md (hunk 1 above)
Comment (conradboucher) [reply README.md]: you're right, i will do that

Comment (conradboucher) [reply README.md]: just made the change @cstolitzka

**Comment by cstolitzka** [comment]
@conradboucher good to merge

**Pull request merged**